boxcryptor vs veracrypt (#27)
worldbank/dime-standards: dime-research-standards/pillar-4-data-security/data-security-resources/veracrypt-guidelines.md

## Why VeraCrypt and not alternatives like BoxCryptor
 
DIME Analytics cannot support all software. 
We select the software that we think is best and 
can be used in the greatest number of contexts 
so as few software as possible needs to be learned by the project teams. 
This does not mean that the software we do not select are all terrible. 
We selected VeraCrypt over competitors as BoxCryptor as we think VeraCrypt is 
more secure, can be used in more context and will always be free to use.

Here are the reasons for why we think VeraCrypt is the better alternative:
 
* BoxCryptor can only be used when securing content in a synced folder in for example DropBox. 
VeraCrypt can also be used for that in addition to other use cases, 
such as encrypting files to be sent over email or encrypting files stored on an external hard drive. 
If you learn VeraCrypt you do not need to learn other encryptions software
for any other use case common in a typical research project. 

> * VeraCrypt is open source and thousands of independent security experts 
> have scrutinized the implementation instead of only a handful who are paid by BoxCryptor.

Reviewer: ```suggestion
* VeraCrypt is open-source has been verified by independent security experts, 
rather than being a proprietary and closed software.
Such tools have had serious security flaws in the past and cannot be independently verified.
```

* VeraCrypt allows a more granular decrypting, 
where you decide exactly what folders are decrypted at each given time. 

> At the time of our research, 
> BoxCryptor decrypts everything you've encrypted once you've entered your decryption key. 

Reviewer: ```suggestion
At the time of this writing, 
Boxcryptor decrypts everything you've encrypted once you've entered your decryption key. 
```

Clarify: everything where? In the selected folder? In the user's account? (I am not sure)

That means that if your computer is compromised, 

> then the intruder gets access to all your encrypted information when using BoxCryptor. 

Reviewer: ```suggestion
then the intruder gets access to all your encrypted information when using Boxcryptor. 
```

> In VeraCrypt only the exact folder or folders you decrypted 
> while you were hacked, exposed to malware, etc., could have been leaked.

Reviewer: ```suggestion
In VeraCrypt, only the exact folder or folders that you intentionally decrypted
during a security breach such as a hack or malware could have been leaked.
```

> * Our understanding is that BoxCryptor has some limits on the free tier 
> and we would have no way of paying for premium tiers if team runs into restrictions. 
> VeraCrypt is open source and will never have cost implications for the team.

Reviewer: ```suggestion
* Proprietary tools like Boxcryptor have functionality limits at free tiers 
and, as with most cloud-based services, cannot easily be extended or paid for within Bank rules
in case additional functionality is needed, requiring a costly retool and potential loss of information
not under the control of the team. 
VeraCrypt is open source and will never have cost implications for the team,
nor will functionality ever be limited or cancelled.
```

> These reasons do not make BoxCryptor a terrible alternative, 
> and it is far superior to using no encryption. 
> What is most important is that your project has a data security protocol 
> that includes proper encryption and that all team members follow it. 

Reviewer: ```suggestion
These reasons do not rule out the use of Boxcryptor, 
and it is far superior to using no encryption,
especially if it is a standard tool for an external partner
or there are other compelling reasons to use such software. 
What is most important is that your project has a data security protocol 
that includes proper encryption and that all team members follow it. 
```
